test/cag-common-transforms.js

```diff
@@ -61,18 +61,7 @@ test('CAG should intersect properly', t => {
   t.deepEqual(compactCagSide(lastSide), {pos: [[0.30901699437494723, -0.9510565162951536], [1, 0]]})
 })
 
-test('CAG should transform properly', t => {
-  t.fail('TODO: add tests') // non implemented tests should fail so we see the changes when we actually add them!!
-})
-
-test('CAG should flip properly', t => {
-  t.fail('TODO: add tests') // non implemented tests should fail so we see the changes when we actually add them!!
-})
-
-test('CAG should expand properly', t => {
-  t.fail('TODO: add tests') // non implemented tests should fail so we see the changes when we actually add them!!
-})
-
-test('CAG should contract properly', t => {
-  t.fail('TODO: add tests') // non implemented tests should fail so we see the changes when we actually add them!!
-})
+test.todo('CAG should transform properly');
```

Comment:
@z3dev I personally prefer to flag the todos with this ava suffix, hope you don't mind.